src/tests/unit/tests/DetailsView/components/__snapshots__/issues-table.test.tsx.snap

```diff
@@ -27,11 +27,13 @@ exports[`IssuesTableTest automated checks disabled 1`] = `
         className="details-disabled-message"
         role="alert"
       >
-        Turn on 
-        <Term>
-          Automated checks
-        </Term>
-         to see a list of failures.
+        <span>
```

Comment:
Should there be a snapshot test that verifies the new text?

Reply:
yeap... I also notices when change the case of 's'